misk-admin/api/misk-admin.api

```diff
@@ -205,12 +205,15 @@ public final class misk/web/dashboard/ConfigDashboardTabModule : misk/inject/KAb
 }
 
 public final class misk/web/dashboard/DashboardHomeUrl : misk/web/dashboard/ValidWebEntry {
-	public fun <init> (Ljava/lang/String;Ljava/lang/String;)V
+	public fun <init> (Ljava/lang/String;Ljava/lang/String;Lkotlin/reflect/KClass;)V
```

Comment:
are these changes backwards compatible?

Reply:
I'll do another review but from the ones I checked; yes: they're either backwards compatible or are unused across all internal repos / public Github.